stream.c

```diff
@@ -153,15 +174,15 @@ static inline bool cobs_encode_stream_single(struct cobs_encode *encode, uint8_t
 	case COBS_ENCODE_STATE_ZEROS_CODE: {
 		if (encode->u.zeros.next_zero == 0) {
 			uint8_t zero;
-			size_t num_pulled = net_buf_pull_across_fragments(&encode->buf, &zero, 1);
-			__ASSERT_NO_MSG(num_pulled == 1);
-			ARG_UNUSED(num_pulled);
+			ret = cobs_buf_cursor_read(&encode->cursor, &zero, 1);
```

Comment:
sizeof(uint8_t value) instead  of magic number (1) ?

Reply:
done